### ISSUE TITLE
3d plot

### ISSUE THREAD

**Issue by michaellevy** [zmjones/rfss#4]
Zach -- I like the 3d plot (interaction.png), but I don't see where you made it. Is that made in R?

**Comment by zmjones** [comment]
yes I think we did it using `plot3D`. I will see if I can find the code snippet somewhere. This paper is kind of dead-in-the-water. I think the `plot3d` function in `rgl` is better though.

**Comment by michaellevy** [comment]
Nice, thanks. The little playing with 3d plots I've done has left me
wanting. I'll check out plot3d.
